sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/RestrictDeserializedTypes.cs

```diff
@@ -51,14 +51,12 @@ internal sealed class RestrictDeserializedTypes : ISymbolicExecutionAnalyzer
 
```

Comment:
pedantic: unused using, there is also an issue regarding this: https://sonarcloud.io/project/issues?id=sonaranalyzer-dotnet&open=AXQ2NZ4-JNYZ1-4E9868&pullRequest=3575&resolved=false&types=CODE_SMELL